### ISSUE TITLE
Unable to install psi4 using conda/mamba

### ISSUE THREAD

**Issue by dww100** [psi4/psi4#2898]
I am trying to install the package using conda/mamba in a fresh environment, I get the following error using `mamba install -c psi4 psi4`:

`nothing provides mpc 1.0.1 needed by gcc-5-mp-5.2.0-0`

Am I doing something wrong or does the feedstock need updating?

**Comment by loriab** [comment]
No reason you should know this, but `gcc-5-mp` is sign that the solver can't install a recent psi4 into the existing environment and so is instead going back 5+ years to a time when packages were less defensively pinned and conda only had gcc v4 available on defaults channel.

I'd suggest creating a new env and specifying the python. If needed, you can specify the psi4 version. And if you've got your setup pulling from conda-forge at higher priority than defaults, you can clobber that, too. So something along the lines of the below, to be over-explicit. Also see https://psicode.org/installs/v17/ if you haven't already. If that works, but you really don't want a separate environment, then try the more explicit install into your working env and/or post back here with your `conda list`, and I'll try to see if there's any red flags.

```
conda create -n p4env python=3.9 psi4=1.7 -c defaults -c psi4
```

**Comment by dww100** [comment]
Thanks so much @loriab - I think the conda-forge over defaults must have been the root cause of the issues (I have tried the conda command from the link you sent and was reluctant to have a separate conda installation)

**Comment by loriab** [comment]
Ah, yes, c-f vs. defaults is a likely cause. There are a couple ways to deal with mixing psi4 and c-f deps, but none that's simple, robust, and a one-liner. Whatever you do, `blas=*=mkl` is a must with c-f and psi. Then you can (1) maintain two channels for the two bases and mix them at the PYTHONPATH level, (2) start an env with a defaults + psi4 install, then install additional packages from c-f, (3) use an env spec file mixes a known safe combination of psi4 and c-f packages.

This is admittedly aggravating for an ordinary request. It should get better soon, as I'm working on c-f packaging (there have been dependency blocks up until about a month ago). I can try to interpret any roadblocks you hit in the meantime.

> reluctant to have a separate conda installation)

Just to be clear, you wouldn't need a separate conda _installation_; only a separate conda _environment_.

**Comment by dww100** [comment]
Thanks for those extra points - really good to know.

On the second point there I was referring to the installer release - does that not create a new installation (hence the lines to alter the bashrc etc in the installation commands)? [I understand my original post was ambiguous/as clear as mud]

**Comment by loriab** [comment]
Yes, you're quite right that the installer release (essentially miniconda with psi4 pre-installed) is a new conda installation. Sorry, for the people willing to type `conda install`, I forget they may have considered the installer. :-)
